tests/functional/settings_clients.js

```diff
@@ -161,8 +161,8 @@ define([
         .waitForDeletedByCssSelector('#client-disconnect')
         .end()
 
-        .then(click('.client-device:nth-child(2) .client-disconnect'))
-        .then(click('select.disconnect-reasons > option[value="lost"]'))
+        .then(click('.client-device:nth-child(1) .client-disconnect'))
```

Comment:
hm did you have to change this line? I think this changes the button you click (there are two in the list)